### PR TITLE
Longhorn:  Update dev-v2.5 for v1.1.2 release

### PR DESCRIPTION
**Pull request by shuo-wu**
https://github.com/longhorn/longhorn/issues/2724

### CODE REVIEW

**Comment by shuo-wu** [comment]
Test: Directly add the branch of my repo to Cluster Explorer Chart Repos.

**Comment by khushboo-rancher** [comment]
Deployment failed with error `ImagePullBackOff: Back-off pulling image "rancher/mirrored-longhornio-longhorn-manager:v1.1.2"`

**Comment by cbron** [comment]
@innobead @shuo-wu  is this going into 2.5.10 or 2.6.0 or both ? Guessing both.

**Comment by jiaqiluo** [comment]
need to update the version in [this issue ](https://github.com/rancher/rancher/issues/33528) once this PR is merged

**Review by jiaqiluo — DISMISSED** [review]
LGTM

**Comment by innobead** [comment]
> @innobead @shuo-wu is this going into 2.5.10 or 2.6.0 or both ? Guessing both. If so after this is merged we need to add same PR to dev-v2.6, but code freeze for that is Tuesday.

@cbron Yes, we need both.

**Comment by khushboo-rancher** [comment]
Validated with the `Shuo-wo` branch after the PR https://github.com/rancher/image-mirror/pull/141 is merged.

Validation - **Pass**
The installation and uninstallation working fine.

File: packages/longhorn/generated-changes/patch/README.md.patch
Comment (nickgerace) [inline packages/longhorn/generated-changes/patch/README.md.patch:3]: Why is there a README patch in general?

File: packages/longhorn/generated-changes/patch/questions.yml.patch
Comment (nickgerace) [inline packages/longhorn/generated-changes/patch/questions.yml.patch:42]: What is this varaible?

File: packages/longhorn/generated-changes/patch/README.md.patch
Comment (shuo-wu) [reply packages/longhorn/generated-changes/patch/README.md.patch:3]: There are **Helm** Chart installation & uninstallation guides in the original README doc. These guides don't apply to this Rancher Chart.

File: packages/longhorn/generated-changes/patch/questions.yml.patch
Comment (shuo-wu) [reply packages/longhorn/generated-changes/patch/questions.yml.patch:42]: Oh. I just realized the original Chart is not up-to-date. Thank you for pointing it out!

**Review by nickgerace — APPROVED** [review]

**Review by jiaqiluo — APPROVED** [review]
LGTM

**Comment by shuo-wu** [comment]
Any updates are required for this PR? Some users are waiting for this feature chart release.

**Comment by jehof** [comment]
will longhorn 1.1.2 also be available in rancher 2.5.8?

**Review by cbron — APPROVED** [review]

**Comment by cbron** [comment]
May need to adjust versioning in the future. Adding for now.

**Comment by cbron** [comment]
@jehof it should be available now via the cluster-manager apps interface. It won't be available through cluster explorer until the next 2.5 release.

**Pull request merged**
